Doc/library/random.rst

```diff
@@ -142,10 +142,11 @@ Functions for integers
 
 .. function:: getrandbits(k)
 
-   Returns a Python integer with *k* random bits. This method is supplied with
-   the MersenneTwister generator and some other generators may also provide it
-   as an optional part of the API. When available, :meth:`getrandbits` enables
-   :meth:`randrange` to handle arbitrarily large ranges.
+   Returns an unsigned, Python integer with *k* random bits. This method is
```

Comment:
Nitpick level comment, but I'd prefer the term "nonnegative" (or "non-negative", depending on your hyphenation preferences) to "unsigned". The terms "unsigned" and "signed" are usually used in reference to integer *types* (e.g. in C), while nonnegative directly describes the value. Python doesn't have unsigned and signed integers (outside ctypes), and I find the "unsigned" here a bit confusing.